openBCIBoard.js

```diff
@@ -1149,8 +1148,8 @@ function OpenBCIFactory() {
             while (readingPosition <= bytesToRead - k.OBCIPacketSize) {
                 if (data[readingPosition] === k.OBCIByteStart) {
                     var rawPacket = data.slice(readingPosition, readingPosition + k.OBCIPacketSize);
-                    this.emit('rawDataPacket',rawPacket);
                     if (data[readingPosition + k.OBCIPacketSize - 1] === k.OBCIByteStop) {
+                        this.emit('rawDataPacket',rawPacket);
```

Comment:
This does not fix the bug i think i found. This PR simply prevents packets being emitted from the bug i found. I might make just make another PR with the 1.0.0 packet parsing method in an update called 0.3.10